flink-kubernetes/src/test/java/org/apache/flink/kubernetes/highavailability/KubernetesStateHandleStoreITCase.java

```diff
@@ -68,8 +68,8 @@ public void testMultipleKubernetesStateHandleStores() throws Exception {
                 new TestingLeaderCallbackHandler[leaderNum];
 
         @SuppressWarnings("unchecked")
-        final KubernetesStateHandleStore<Long>[] stateHandleStores =
-                new KubernetesStateHandleStore[leaderNum];
+        final KubernetesStateHandleStore<TestingLongStateHandleHelper.LongStateHandle>[]
```

Comment:
The Kubernetes-related ITCases are currently not running. I created [FLINK-20564](https://issues.apache.org/jira/browse/FLINK-22564) to cover this. I haven't had the chance to run the ITCase locally, yet.